service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -60,7 +64,7 @@
         FormatStyle.MEDIUM).localizedBy(Locale.UK);
 
     @Test
-    void aboutToStartCallbackShouldPopulateHearingDatesList() throws Exception {
+    void aboutToStartCallbackShouldPopulateHearingDatesListAndRemoveAllPartiesCustomDirection() throws Exception {
```

Comment:
I would consider updating this test name, it asserts that all custom role collections are not in caseData